package.json

```
@@ -46,7 +39,14 @@
     ]
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
```

I forgot to mention two more to be moved to **devDependencies**
```suggestion
"react-scripts": "5.0.1",
"typescript": "^4.9.5",
```

BTW you can check also this if interested https://github.com/facebook/create-react-app/issues/10219#issuecomment-761613906